src/components/side/Side.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import Stegindikator from 'nav-frontend-stegindikator/lib/stegindikator';
```

Comment:
Hvorfor har denne fått rar import?

Reply:
Er alltid to imports som dukker opp i intellij på de nav-frontend komponentene 🤷‍♀  
Denne har sikkert bare klart å snike seg inn 😌  
Fiksa det!